docs/reference/release-notes/highlights.asciidoc

```diff
@@ -24,53 +24,79 @@ Other versions:
+[[exclude_clusters_from_cross_cluster_search]]
+=== Exclude clusters from a cross-cluster search
+When doing a cross-cluster search, if you use a wildcard to include a list of clusters, you
+can explicitly exclude one or more clusters with a minus sign ('-'). To exclude an entire cluster,
+put the minus sign in front of the cluster alias, such as: `remote*:*,-remote3:*,-remote5:*`.
+In that case, clusters with the alias "remote3" and "remote5" will be excluded from the search.
+For this feature, you must use `*` in the index position or an error will be returned.
 
-{es-pull}96224[#96224]
+{es-pull}97865[#97865]
 
 [discrete]
-[[make_text_embedding_for_knn_search_ga]]
-=== Make text embedding for kNN search GA
-From 8.9, the `text_embedding` `query_vector_builder` extension to kNN search is generally available. This functionality is required to perform <<semantic-search,semantic search>> by converting text to dense vectors.
+[[enable_parallel_knn_search_across_segments]]
+=== Enable parallel knn search across segments
+Elasticsearch has until now performed search sequentially across the
+segments within each shard. This change makes knn queries faster on shards
+that are made of more than one segment, by rewriting and collecting each
+segment in parallel.
 
-{es-pull}96735[#96735]
+{es-pull}98204[#98204]
 
 // end::notable-highlights[]
 
 
 [discrete]
-[[asset_tracking_geo_line_in_time_series_aggregations]]
-=== Asset tracking - geo_line in time-series aggregations
-The <<search-aggregations-metrics-geo-line,`geo_line` aggregation>> builds tracks from `geo_points`.
-It has previously needed to use large arrays in memory for collecting points into multiple buckets
-and sorting those buckets.
-
-With the advances made in TSDB features and the `time_series` aggregation in particular,
-it is now possible to rely on data aggregating in both TSID and timestamp order,
-enabling the removal of all sorting, as well as the use of only a single bucket's
-worth of memory, a dramatic improvement in memory footprint. In addition, we can use the streaming line
-simplifier algorithm introduced in https://github.com/elastic/elasticsearch/pull/94859 to replace the previous
-behaviour of truncating very large tracks with the far more preferable approach of simplifying those tracks.
-
-[role="screenshot"]
-image:images/spatial/kodiak_geo_line_simplified.png[North short of Kodiak Island simplified to 100 points]
-
-In this diagram, the grey line is the original geometry, the blue line is the truncated geometry as would be
-produced by the original `geo_line` aggregation, and the magenta line is the new simplified geometry.
-
-{es-pull}94954[#94954]
+[[change_pre_configured_cached_analyzer_components_to_use_indexversion_instead_of_version]]
```

Comment:
```suggestion
[[change_pre_configured_cached_analyzer_components_to_use_indexversion_instead_of_version-highlight]]
```

Reply:
This is a duplicate ID that would break the docs (it also appears in the migration guide). I think CI didn't catch it because the 8.10 docs do not exist yet.